projectfiles/Xcode/Wesnoth.xcodeproj/project.pbxproj

```diff
@@ -6175,21 +6241,12 @@
 				);
 				INFOPLIST_FILE = Info.plist;
 				INSTALL_PATH = "$(HOME)/Applications";
-				LIBRARY_SEARCH_PATHS = (
```

Comment:
This looks a little suspicious. What did you change that could've deleted this? If it's just removal of an override setting for a specific target (with the resultant setting remaining the same), then that's fine...

Reply:
This is duplicate override of same thing. Library search paths are defined globaly for whole project on line 6335 (DEBUG) and 6387 (RELEASE). This overrides it with same value so it is useless.